fe1-web/src/__tests__/utils/TestUtils.ts

```diff
@@ -72,3 +73,14 @@ export const mockJsonRequest: Partial<JsonRpcRequest> = {
 };
 
 export const mockJsonResponse: Partial<JsonRpcResponse> = { id: 0, result: [] };
+
+export function wrapWithClearMockStorageReducer(reducers: Reducer): Reducer {
+  return (state: any, action: AnyAction) => {
+    let newState = state;
+
+    if (action.type === 'CLEAR_STORAGE') {
+      newState = undefined;
+    }
+    return reducers(newState, action);
+  };
+}
```

Comment:
To be honest, once I saw this I realized that I might have been wrong when asking you to declare the store before the tests (right after the `describe`). I think it was cleaner and more readable to create a new one for each test (it was in `LaoProperties.test.ts`). Wrapping it as you did is fine, but it seems more complicated to me.

Reply:
Yes I agree that it is simpler to create a new one for each test :) Should I leave `wrapWithClearMockStorageReducer` in the codebase or remove it again?

Reply:
As you wish, but I guess it won't be used anywhere else for now, so you can delete it.